datastore/nachet/__init__.py

```diff
@@ -694,21 +694,31 @@ async def get_picture_sets_info(cursor, user_id: str):
         raise  picture.GetPictureSetError(f"An error occured while retrieving the picture sets : {e}")
 
 
-async def get_picture_inference(cursor, user_id: str, picture_id: str):
+async def get_picture_inference(cursor, user_id: str, picture_id: str = None, inference_id :str = None):
```

Comment:
why do we need the inference_id if we have the picture_id. I think you are miss using this function? From my understanding the goal of it is to fetch an inference from a picture_id. if you already have the inference_id, you shouldn't need the picture_id.

Reply:
I want to retrieve the updated inference after a feedback to update the view on the frontend https://github.com/ai-cfia/nachet-backend/issues/119
But during the feedback I only have the inference_id.
I think the goal of this function is to retrieve the inference of a picture so we might as well use it in this case, what do you think ?